apps/account-api/src/services/delegation.service.ts

```diff
@@ -36,4 +47,23 @@ export class DelegationService {
     }
     throw new Error('Invalid msaId.');
   }
+
+  async getRevokeDelegationPayload(accountId: string, providerId: string): Promise<RevokeDelegationPayloadResponseDto> {
+    // TODO: Validate input: make sure accountId and providerId are valid and the delegation exists
```

Comment:
Finish TODO

Reply:
I recommend the following responses:

* either `accountId` or `providerId` does not exist as an MSA => `HttpStatus.NOT_FOUND`
* `providerId` is a valid MSA but not a registered provider => `HttpStatus.BAD_REQUEST`
* no delegation found for account/provider pair => `HttpStatus.NOT_FOUND`